helpers/copyUnstableIncludedDirs.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path')
+const AdmZip = require('adm-zip')
+const getNetlifyFunctionName = require('../src/lib/helpers/getNetlifyFunctionName')
+
+// This is UNSTABLE support for special use cases needing files served with their Next.js
+// pages that become Netlify functions. Eventually this will be supported internally.
+const copyUnstableIncludedDirs = ({ nextConfig, functionsDist }) => {
+  for (const name in nextConfig.unstableNetlifyFunctionsSupport || {}) {
+    const includeDirs = nextConfig.unstableNetlifyFunctionsSupport[name].includeDirs || []
+    console.log('Processing included dirs for ', name)
+
+    const zipName = path.join(functionsDist, getNetlifyFunctionName(name) + '.zip')
+    const zip = new AdmZip(zipName)
+    includeDirs.forEach((includes) => {
+      if (fs.lstatSync(includes).isDirectory()) {
```

Comment:
`require('fs')` is missing.

Note: we might want [to add `eslint`](https://github.com/netlify/eslint-config-node) to prevent this?

Reply:
https://github.com/netlify/netlify-plugin-nextjs/issues/210 :D